.devfile

```diff
@@ -0,0 +1,24 @@
+specVersion: 0.0.1
+name: che-in-che
+projects:
+  - name: che
+    source:
+      type: git
+      location: 'https://github.com/eclipse/che.git'
+tools:
+  - name: theia-editor
+    type: cheEditor
+    id: org.eclipse.che.editor.theia:1.0.0
+  - name: exec-plugin
+    type: chePlugin
+    id: che-machine-exec-plugin:0.0.1
+  - name: che-dev
+    type: openshift
```

Comment:
This doesn't work on kubernetes. If you set `type: kubernetes` would that work on OpenShift too?

Reply:
@l0rd Good catch! `kubernetes` tool works fine on both of Kuberentes/OpenShift infrastructures since `.che-dev.yaml` does not contain OpenShift specific objects like Routes.